manager/manager.py

```diff
@@ -598,7 +598,7 @@ def generate_dashboard(self, max_failures=MAX_FAILURES):
                 '<th class="hdr">Repo</th>'
                 '<th class="hdr">PreConvert</th>'
                 '<th class="hdr">Converted</th>'
-                '<th class="hdr">Destination</th>'
+                '<th class="hdr">Destination</th>',
```

Comment:
I disagree. The missing comma is just a symptom - the real problem is code that appears to do one thing but actually does something else. The way it's currently written it looks like the `BeautifulSoup` constructor takes 8 parameters. The use of EISC (evil implicit string concatenation) inside a parameter list is the cause, and the missing comma is a symptom that points to that cause.